### ISSUE TITLE
Additional information associated with some form fields are not available to screen readers

### ISSUE THREAD

**Issue by jose-cds** [cds-snc/platform-forms-client#3090]
**HC Assessment Label: Serious**

Pages:  Advancing Accessibility Standards Research Program Expression of Interest
URL: https://forms-staging.cdssandbox.xyz/en/id/134
User agent:  Chrome (v119.0.6045.200), MS Edge (120.0.2210.77), JAWS2020, NVDA 2023.1
Language: English and French

Issue Description: 
Additional information or instruction conveyed visually above some form fields which can be an important cue or instruction for filling out the fields accurately. In this case, it is not provided programmatically for assistive technology users. When additional information is not associated with the form field, screen reader users will miss the information when they tab through the focusable elements in focus mode to interact with the form. For example, see attached screenshot.

Steps to reproduce:
1. Run the NVDA screen reader
2. Open the above URL in the Chrome browser
3. Navigate using the "Tab" key

Expected result: The additional information should be associated programmatically with the corresponding form controls using an appropriate method to ensure it is available to tscreen readers in focus mode.
Any instructions that are common to all form fields or users should know in advance to fill out the form correctly could be provided at the top of the form. This needs to be addressed based on each form.

**Comment by srtalbot** [comment]
Fixed, we removed the placeholder text. Only description text is available now, which screen readers can read.
